### ISSUE TITLE
Page Load Completed

### ISSUE THREAD

**Issue by msawlorSDI** [searchdiscovery/Apollo-Documentation-Connected-Vehicle#3]
[Page Load Completed](https://github.com/searchdiscovery/Apollo-Documentation-Connected-Vehicle/blob/1f284b5b3e98be9636ce29d275b85c2c736aacad/Data%20Layer%20Events/Page%20Load%20Completed.md)

**Comment by jeffkush-sdi** [comment]
PAGE LOAD COMPLETE STATUS: Event order/bracketing may need further discussion (see #2 Page Load Started). QA was only for datalayer structure/order @ this point in time.

https://cwp.sit.siriusxmguardian.com/manage
Page Load Completed event should be fired as the last event on any page/virtual page load, with regards to any events that need/should be bracketed. There would be no expectation of seeing a Page Load Completed event fire AFTER a ex. Product Added to Cart event, since this event does not need to be bracketed.

Can SXM team confirm that the current implementation for Page Load Completed is essentially to fire the 'completion event' on the next page? thx

![Image](https://github.com/searchdiscovery/Apollo-Documentation-Connected-Vehicle/assets/116372762/ccdd268a-a2e4-461b-b36d-4cfc91fa4e16)

_(see page load started for deeper discuss on bracketed events)_

**Comment by Shocopop** [comment]
@jeffkush-sdi There is still one open question: in case of `Product Listing Displayed` -> that event can be sent multiple times (every time a user switches from "monthly" to "annual" tab). In that case, we cannot close the bracket until the user progresses to the next screen. There is no way around it unless we open and close a bracket every time for each `Product Listing Displayed` event. Out of those two options, which one is preferable?
![image](https://github.com/searchdiscovery/Apollo-Documentation-Connected-Vehicle/assets/23707589/1cd2b3e7-1012-4689-9480-7f9db5fdb1d9)

**Comment by jeffkush-sdi** [comment]
@Shocopop

Yes the **Product Listing Displayed** event should be fired after each 'product listing display'  _change_; by bracketing this event in-between 'page load start/completed', each time/instance, the event needs to be fired (same page context...ie essentially this creates a virtual page view hit). cc @msawlorSDI

**Comment by jeffkush-sdi** [comment]
@Shocopop 

### 'Page Load Completed' DataLayer QA:

Use Case: https://cwp.sit.siriusxmguardian.com/manage;  _other use cases to follow, where applicable_

![Image](https://github.com/searchdiscovery/Apollo-Documentation-Connected-Vehicle/assets/116372762/8f14f92b-346a-4deb-8637-3bcbc47bd855)

**Datalayer event schema looks correct and the event does appear to be bracketed with a opening Page Load Started event in the 'same page context'. Additionally, (@ snapshot in time), the event is being fired AFTER the appropriate/applicable event(s)**

**[FWW] To Review Server Calls/Rules...**

cc @msawlorSDI
